lib/main.dart

```diff
@@ -12,6 +12,8 @@ import 'view/info_view.dart';
 import 'view/intubation/algorithm/intubation_algorithm_page.dart';
 import 'view/intubation/checklist/intubation_checklist_page.dart';
 import 'view/intubation/guidance/intubation_guidance_page.dart';
+import 'view/ppe/ppe_off_guidance_1.dart';
+import 'view/ppe/ppe_off_guidance_2.dart';
```

Comment:
personally, I prefer import with explicit package link

Reply:
@suesitran @domesticmouse pointed out in a different PR that relative imports within packages  is actually the recommended way in the Dart style guide and he has added a lint to enforce that usage, so this isn't a matter of develper personal preference.